accord-core/src/main/java/accord/messages/Accept.java

```diff
@@ -241,10 +239,12 @@ public void process()
         @Override
         public AcceptReply apply(SafeCommandStore safeStore)
         {
-            SafeCommand safeCommand = safeStore.command(txnId);
+            SafeCommand safeCommand = safeStore.get(txnId, someKey);
             switch (Commands.acceptInvalidate(safeStore, safeCommand, ballot))
             {
                 default:
```

Comment:
default is truncated?  maybe we should fail?

```suggestion
                default:  throw new IllegalArgumentException("Unknown status: " + outcome);
```